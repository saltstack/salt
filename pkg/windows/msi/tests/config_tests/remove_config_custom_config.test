properties CONFIG_TYPE=Custom CUSTOM_CONFIG=tests\config_tests\remove_config_custom_config.conf START_MINION="" REMOVE_CONFIG=1
