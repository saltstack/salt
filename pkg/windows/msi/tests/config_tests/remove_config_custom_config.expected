# User config line 1/3
master: custom.master
# User config line 2/3
id: custom.minion
# User config line 3/3
